### PR TITLE
Bug 1115622 - [Bluetooth] decouple bluetoothTransfer and NfcHandoverMana... +autoland

### PR DESCRIPTION
**Pull request by gasolin**
...ger's dependency

### CODE REVIEW

**Comment by try-server-hook** [comment]
[gasolin gasolin (gasolin)](https://github.com/gasolin) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8a4d42039033)

File: apps/system/js/bluetooth_transfer.js [hunk 1]
```diff
@@ -122,7 +117,7 @@ var BluetoothTransfer = {
     });
   },
 
-  onFilesSending: function bt_onFilesSending(evt) {
+  _onFilesSending: function bt_onFilesSending(evt) {
```
Comment (ian-liu) [inline apps/system/js/bluetooth_transfer.js]: nit: `_onFilesSending: function bt__onFilesSending(evt) {`

File: apps/system/js/bluetooth_transfer.js [hunk 2]
```diff
@@ -330,7 +325,7 @@ var BluetoothTransfer = {
     }
   },
 
-  onUpdateProgress: function bt_onUpdateProgress(mode, evt) {
+  _onUpdateProgress: function bt_onUpdateProgress(mode, evt) {
```
Comment (ian-liu) [inline apps/system/js/bluetooth_transfer.js]: nit: `_onUpdateProgress: function bt__onUpdateProgress(mode, evt) {`

File: apps/system/js/bluetooth_transfer.js [hunk 3]
```diff
@@ -452,7 +447,7 @@ var BluetoothTransfer = {
     }
   },
 
-  onTransferComplete: function bt_onTransferComplete(evt) {
+  _onTransferComplete: function bt_onTransferComplete(evt) {
```
Comment (ian-liu) [inline apps/system/js/bluetooth_transfer.js]: same here

File: apps/system/js/nfc_handover_manager.js [hunk 4]
```diff
@@ -664,7 +668,7 @@ var NfcHandoverManager = {
    * BluetoothTransfer notifies us that a file transfer has started.
    * @memberof NfcHandoverManager.prototype
    */
-  transferStarted: function transferStarted() {
+  _transferStarted: function transferStarted() {
```
Comment (ian-liu) [inline apps/system/js/nfc_handover_manager.js]: same here

File: apps/system/test/unit/nfc_handover_manager_test.js [hunk 5]
```diff
@@ -47,9 +47,14 @@ suite('Nfc Handover Manager Functions', function() {
     realMozSetMessageHandler = navigator.mozSetMessageHandler;
     realL10n = navigator.mozL10n;
 
+    Object.defineProperty(navigator, 'mozBluetooth', {
+      configurable: true,
+      get: function() {
+        return MockBluetooth;
+      }
+    });
```
Comment (ian-liu) [inline apps/system/test/unit/nfc_handover_manager_test.js]: Good catch here. :100:

**Comment by try-server-hook** [comment]
[gasolin gasolin (gasolin)](https://github.com/gasolin) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=777b3cc53c09)

**Pull request merged**